pyproject.toml

```diff
@@ -6,3 +6,7 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools_scm]
 # See configuration details in https://github.com/pypa/setuptools_scm
 version_scheme = "no-guess-dev"
+
+[tool.pylint.'MESSAGES CONTROL']
+max-line-length = 120
+disable = " R0801, R0902, R0913, R0914"
```

Comment:
It's better to use the names instead of the codes, because most humans don't autotranslate

Reply:
Good point, Ill convert them to names